crates/joystick/src/windows.rs

```diff
@@ -16,62 +8,41 @@ enum KeyAction {
     Release,
 }
 
+pub struct Joystick {
+    device: Xbox360Wired<Client>, // may need to be Arc<Mutex>>
+    gamepad: vigem_client::XGamepad,
+    events: Vec<JoystickEvent>,
+    mask: u16, // current mask for the joystick
```

Comment:
What is the mask specifically? Which buttons are pressed?

Reply:
yes - renaming it to button_mask